### ISSUE TITLE
PreferredMediaType?

### ISSUE THREAD

**Issue by wkearney99** [adultmm/AdultMediaManager#91]
In the AE movie scraper, what does this setting do?  It has two choices: DVD and BluRay.  

When set to DVD and using "Rescrape, Skip if more than one match" it doesn't always seem to be able to pick DVD when there's just DVD and VOD options for a title.

Should it also have a VOD choice in addition to DVD and BluRay?  And is that the right spelling, instead of "Blu-Ray"?

The goal here is to batch Rescrape and have it default to matching to the selected media type.  Or is that not what this setting controls?

**Comment by adultmm** [comment]
With the DVD option, AMM prefers DVD movies instead of Bluray releases, IF the movies have the same matching score, calculated from their name. The aspect ratio of DVD cover is better for most of the media centers, but some users prerfers Bluray covers.
If you think AMM detects some movies incorrectly, send me the examples, and I'll investigate it.

**Comment by wkearney99** [comment]
An example of this is scraping movies from the "Natural Beauties" series.  There's a DVD and a VOD entry for each of them.  It'd be handy of the scraper could be defaulted to using one or the other.  Use the "[Natural Beauties Vol. 6](https://www.adultempire.com/1950977/natural-beauties-vol-6-porn-movies.html)" title as an example.  

This holds true for many other movies on AE.

**Comment by adultmm** [comment]
Do I understand correctly that your issue is that if the same movie is available in both DVD and VOD formats, the scraper does not automatically select one of them, even if the PreferredMediaType is set?

**Comment by wkearney99** [comment]
Yes, that's correct.  It'd be convenient to be able to tell it to use one or the other when doing automatic searches.  I don't know if there's more than DVD and VOD as options, I've only seen those two.

**Comment by adultmm** [comment]
The tricky part is detecting the same movie — there are different movies with the exact same title and year.
